xcode/Subconscious/Shared/Components/Common/Profile/UserProfileView.swift

```diff
@@ -192,16 +186,30 @@ struct UserProfileView: View {
                             .init([.top, .horizontal]),
                             AppTheme.padding
                         )
+                    } else {
+                        ProfileHeaderPlaceholderView()
                     }
                     
-                    TabbedTwoColumnView(
-                        columnA: columnRecent,
-                        columnB: columnFollowing,
-                        selectedColumnIndex: state.currentTabIndex,
-                        changeColumn: { index in
-                            send(.tabIndexSelected(index))
-                        }
-                    )
+                    if let recent = state.recentEntries,
+                       let following = state.following {
```

Comment:
Instead of hiding the content during loading, lets allow loading in the background and reflect it in the omnibar so the user can tell work is happening while still getting to view the (possibly stale) content.